package.json

```diff
@@ -3,53 +3,51 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
-  "packageManager": "yarn@1.22.19",
-  "workspaces": {
-    "packages": [
-      "packages/*"
-    ],
-    "nohoist": [
-      "**/@types/**",
-      "**/typedoc",
-      "**/typedoc-plugin-markdown",
-      "**/typedoc-plugin-mdn-links"
-    ]
-  },
+  "packageManager": "pnpm@7.24.2",
   "dependencies": {
-    "lerna": "3.16.4"
+    "lerna": "5.6.2"
   },
   "devDependencies": {
+    "@types/node": "14.18.33",
     "@typescript-eslint/eslint-plugin": "5.21.0",
     "@typescript-eslint/parser": "5.21.0",
+    "@vercel/build-utils": "workspace:5.7.5",
+    "@vercel/fun": "1.0.4",
```

Comment:
I think `@vercel/fun` should be in `packages/cli/package.json` instead

Reply:
This chunk somehow needs it at the root level: 

https://github.com/vercel/vercel/blob/4c3bc0532216f21f354623ada76b2315ef8ba519/packages/cli/scripts/build.ts#L69-L76

(Remove, run install, run build, and you'll see the full error)

Reply:
@Ethan-Arrowood Pretty sure that path was written that way because of yarn hoisting. So you can move `@vercel/fun` to the cli package and then update the `build.ts` to read the new path from `./node_modules/@vercel/fun/dist/src/runtimes` I believe

Reply:
ill give that a shot

Reply:
seems to build fine!